privaterelay/templates/settings.html

```diff
@@ -0,0 +1,104 @@
+{% extends "base.html" %}
+
+{% load static %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+{% block javascript %}
+  <script src="{% static 'js/settings.js' %}"></script>
+{% endblock %}
+
+
+{% block content %}
+
+<main id="settings-main" class=" bg-light dashboard-container " data-api-token="{{ user_profile.api_token }}">
+  <div class="c-meta">
+    <h1 class="mzp-u-title-md c-settings-headline">{% ftlmsg 'settings-headline' %}</h1>
+    <ul class="c-meta--links">
+      <li>
+        <a
+          href="https://support.mozilla.org/"
+          target="_blank"
+          rel="noopener noreferrer"
+          title="{% ftlmsg 'settings-meta-help-tooltip' %}"
+          class="glocal-fxa-settings-link"
+        >
+        <img src="/static/images/help.svg" alt="">
+          {% ftlmsg 'settings-meta-help-label' %}
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://status.relay.firefox.com/"
+          target="_blank"
+          rel="noopener noreferrer"
+          title="{% ftlmsg 'settings-meta-status-tooltip' %}"
+          class="glocal-fxa-settings-link"
+        >
+        <img src="/static/images/performance.svg" alt="">
+          {% ftlmsg 'settings-meta-status-label' %}
+        </a>
+      </li>
+    </ul>
+  </div>
+  <div class="c-settings">
+    <div id="save-error" class="hidden c-settings-form--notification">
```

Comment:
Global conversation: The `--` line isn't in line with our current naming convention. I mention it bc I know it's a known pattern in [BEM naming](http://getbem.com/naming/). 

Is this something we should revise to match everything else? I don't have a hard opinion about it, but there's only a [few minor instances](https://github.com/mozilla/fx-private-relay/blob/main/static/scss/components/buttons.scss#L12-L22) (that I wrote 🙈)  of that naming convention elsewhere in the code base. I would like to stick to convention, unless there's a strong reason to deviate.  

_(Flagging this instance here only) _

<img width="321" alt="image" src="https://user-images.githubusercontent.com/2692333/135925069-e43e708a-583f-48ed-839f-36a7ec374438.png">

Reply:
Heh, I was wondering about that - just wasn't sure what the convention then is. Just more `c-` classes? (Maybe you could suggest a name for this specific class - I can then probably deduce appropriate class names for the other instances from that.)